### DISCUSSION TITLE
Issue with Abnormal Data Display

### DISCUSSION THREAD

**Discussion by IanMoone-J** [apache/skywalking#12698] [Q&A]
While using SkyWalking, abnormal data is displayed in the UI, affecting the overall data presentation. Could you please explain why this happens and how I should troubleshoot it?
![image](https://github.com/user-attachments/assets/b28d13ad-0f05-4878-a8f5-961c19b159ed)
There are some error logs in the OAP server, but the timestamps do not match the time when the data anomaly occurred.
![image](https://github.com/user-attachments/assets/d64e79fd-a48d-420d-9cfc-dd5d9005c847)
I look forward to your response.

**Comment by wu-sheng** [comment]
One side, oap logs don't mean something wrong happened. It is some clients(agents or otel collector or something) disconnected from oap.

And those peaks are your application latency issues, not OAP issues.

**Reply by IanMoone-J** [reply]
Thank you for your response.
